### DISCUSSION TITLE
remove DHCP Server settings

### DISCUSSION THREAD

**Discussion by paulnl** [AdguardTeam/AdGuardHome#3349] [Q&A]
Hello all,

I tried to set up the DHCP server now after about 10 devices I had an error and all my DHCP settings were gone, all the devices I had assigned an ip seemed to be gone. Now when I try to set these parameters again I get an error that it already exists. How can I remove the DHCP settings I have no idea where they are in the AdGuardHome folder

**Comment by ainar-g** [comment]
Hello.  The DHCP server settings are in the configuration file, [`AdGuardHome.yaml`].  The leases are stored in a separate file, `leases.db`.

[`AdGuardHome.yaml`]: https://github.com/AdguardTeam/AdGuardHome/wiki/Configuration#configuration-file
